pkg/controller/direct/secretmanager/secret_mapping.go

```diff
@@ -96,8 +103,7 @@ func SecretManagerSecretSpec_ToProto(mapCtx *direct.MapContext, in *krm.SecretMa
 	}
 	// MISSING: Etag
 	out.Rotation = Rotation_ToProto(mapCtx, in.Rotation)
-	// MISSING: VersionAliases
-	// out.VersionAliases = in.VersionAliases
+	out.VersionAliases = MapStringString_ToMapStringInt64(mapCtx, in.VersionAliases)
```

Comment:
Very interesting case! I can do a follow up PR to improve this. This is less likely to happen for SecretManager because the int64 is used a version counts starting from 1.